src/savegame_legacy.cpp

```diff
@@ -1907,7 +1907,7 @@ void item::load_legacy(std::stringstream & dump) {
         }
     }
 
-    dump >> burnt >> poison >> ammotmp >> owned >> bday >>
+    dump >> burnt >> poison >> ammotmp >> bday >>
```

Comment:
In other words you need to write the value to a temporary variable and then
ignore it.